fdbclient/NativeAPI.actor.h

```diff
@@ -289,6 +289,21 @@ class Transaction : NonCopyable {
 		                reverse);
 	}
 
+	[[nodiscard]] Future<RangeResult> getRangeAndHop(const KeySelector& begin,
```

Comment:
Can you add some documentation how to use this API? Especially how to set `hopInfo`?

Reply:
Yeah, there should be a document. But I'm not going to add it for now because the interface is going to be changed very soon in next PR.